djstripe/admin/forms.py

```diff
@@ -45,10 +46,56 @@ def __init__(self, *args, **kwargs):
             )
 
 
-class APIKeyAdminCreateForm(forms.ModelForm):
+class APIKeyAdminBaseForm(forms.ModelForm):
     class Meta:
         model = models.APIKey
-        fields = ["name", "secret"]
+        fields = ["name", "djstripe_is_account_default"]
+
+    @property
+    def construct_custom_error_message(self):
```

Comment:
Make this a separate helper function that takes the type and the account as arguments, instead of a class member.